base_wamas_ubl/lib/wamas/test_utils.py

```diff
@@ -29,6 +41,30 @@ def testDict2wamas(self):
             output = dict2wamas(ast.literal_eval(str_dict_input), telegram)
             self.assertEqual(output, expected_output)
 
+    def testGetAddressNamePicking(self):
+        infile = file_open(
+            file_path("tests/samples/UBL2WAMAS-SAMPLE_AUSK_AUSP-DESPATCH_ADVICE.xml")
+        ).read()
+        dict_item = Dotty(xmltodict.parse(infile))
+        address_elements = get_address_elements(dict_item)
+        self.assertEqual(
+            address_elements,
+            {
+                "ContactName": "Chester Reed",
+                "PartyName": "YourCompany",
+                "Department": "Department_1",
+                "StreetName": "250 Executive Park Blvd, Suite 3400",
+                "AdditionalStreetName": "AdditionalStreetName_1",
+            },
+        )
+        self.assertEqual(get_Adrs_Name(address_elements), "Chester Reed")
+        self.assertEqual(get_Adrs_Name2(address_elements), "YourCompany")
+        self.assertEqual(get_Adrs_Name3(address_elements), "Department_1")
+        self.assertEqual(
+            get_Adrs_Name4(address_elements), "250 Executive Park Blvd, Suite 3400"
+        )
+        self.assertEqual(get_Adrs_Adr(address_elements), "AdditionalStreetName_1")
+
```

Comment:
can you add one more test case not based on a xml file, but based on a given {}
that matches the example given in the specs:  Duffard Pascal
https://docs.google.com/document/d/1JzXZOgPSkvVYP9naE9qA_wIY2T9ElbArvN71f_4cbik/edit

the goal is to have a different example to see different expectations